apps/settings/src/views/Users.vue

```diff
@@ -92,7 +81,32 @@
 						</template>
 					</NcAppNavigationItem>
 
-					<NcAppNavigationCaption v-if="groupList.length > 0" :name="t('settings', 'Groups')" />
+					<!-- TODO remove class -->
+					<NcAppNavigationCaption class="caption"
+						:name="t('settings', 'Groups')"
+						:disabled="loadingAddGroup"
+						:aria-label="loadingAddGroup ? t('settings', 'Creating group…') : t('settings', 'Create group')"
```

Comment:
I'm not sure, but ше seems that we usually add a space before `…`, don't we?

```suggestion
						:aria-label="loadingAddGroup ? t('settings', 'Creating group …') : t('settings', 'Create group')"
```